puppet/modules/redmine/manifests/init.pp

```diff
@@ -126,7 +126,7 @@
   }
 
   exec { 'install redmine':
-    command     => 'bundle install',
+    command     => 'bundle install --path ./vendor',
```

Comment:
Technically `--path` is deprecated and `bundle config --local path ./vendor` + `bundle install` is the right way. Or write down the `.bundle/config` file explicitly.

Reply:
technically, yes. it worked tho, and was the path of lowest resistance ;-)

Reply:
do you have a preferred way to do it? (I'd guess a `.bundle/config` file is cleaner for a cfgmgmt system)

Reply:
I'd think `.bundle/config` indeed, but I'd also accept a `TODO` comment here ;)

Reply:
TODO it is!